src/runtime/lpython_builtin.py

```diff
@@ -858,6 +858,15 @@ def _lpython_str_isascii(s: str) -> bool:
             return False
     return True
 
+@overload
+def _lpython_str_isspace(s:str) -> bool:
+    ch:str 
```

Comment:
Minor: I would add a space between the variable and the annotation.
```suggestion
    ch: str 
```

Reply:
@Agent-Hellboy This way it is in sync with the other existing declarations in the same file.